<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showtask</class>
 <widget class="QDialog" name="showtask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">showtask
{
	background-color:#2D142C;
}
QPushButton
{
	background-color:#801336;
	border-radius:12px;
	border:2px solid #510A32;
}
QPushButton:hover
{
	color: #FFFFFF;
	background-color: #510A32;
}
QLabel
{
	color:#C72C41;
}
QPlainTextEdit
{
	background-color:#510A32;
	color:#C72C41;
	border-radius:12px;
}
QCheckBox
{
	color:#C72C41;
}
QCheckBox:hover
{
	background-color:#C72C41;
	color:#000000;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>729</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout_buttons">
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_edit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_delete">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_priority">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>40</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>48</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>729</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_group">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_group">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_head">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="pte_title">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_description">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_date">
    <item>
     <widget class="QLabel" name="label_createdate">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_createdate">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_deadline">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Deadline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_deadline">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>731</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_note">
    <item>
     <widget class="QLabel" name="label_note">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_note">
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>620</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>722</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_steps">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="cb_step1">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Step1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_step2">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_step3">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_step4">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_step5">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_step6">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stup6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
